### ISSUE TITLE
Missing online accounts

### ISSUE THREAD

**Issue by bil-elmoussaoui** [numixproject/numix-icon-theme#979]
There's a few missing icons for Gnome online accounts:
- [ ] im-google-talk 
- [ ] im-irc
- [ ] im-myspace 
- [ ] im-facebook -> facebook
- [ ] goa-account-foursquare

The im-* icons are used by Empathy. There's more icons, i don't know if its good to add them or not?
![screenshot from 2016-01-01 02-15-06](https://cloud.githubusercontent.com/assets/7660997/12069401/843568f0-b02d-11e5-861b-f7028377d087.png)

**Comment by bil-elmoussaoui** [comment]
Btw: im-jabber need to be removed from Numix Circle..

**Comment by wa4557** [comment]
Google talk is EOL a long time ago...

**Comment by bil-elmoussaoui** [comment]
@wa4557 i didn't knew that..i will remove it from the list

**Comment by wa4557** [comment]
`goa-account` is it a good idea to have it transparent like in `hicolor`?

**Comment by Foggalong** [comment]
Google Talk might be EOL but a few programs still use that name when talking about hangouts. Pidgin is one, I wouldn't be surprised if GOA does the same

**Comment by bil-elmoussaoui** [comment]
@Foggalong yeah, should i add the whole list of im-* icons? some of them are used by Pidgin too

**Comment by Foggalong** [comment]
Yeah, stick em all in :)

**Comment by bil-elmoussaoui** [comment]
@wa4557 Yeah, we can change that later if gnome changes the icon in the future 
@Foggalong Done

**Comment by Foggalong** [comment]
:+1:

**Comment by wa4557** [comment]
@bil-elmoussaoui `im-yahoo` is already there

**Comment by bil-elmoussaoui** [comment]
@wa4557 Done

**Comment by MrTimscampi** [comment]
I figured I'd mention it here instead of opening another issue, since it's related, but the mail-unread-symbolic icon not being centered looks weird in Online Accounts:

![screenshot from 2017-04-19 15-32-27](https://cloud.githubusercontent.com/assets/19396809/25182596/6d415a56-2515-11e7-936f-e1e0f4614d68.png)

According to the GTK inspector, the array of icons is `["mail-unread-symbolic", "mail-symbolic", "mail-unread", "mail"]`.

**Comment by Foggalong** [comment]
@MrTimscampi Did you upload the wrong screenshot? I'm confused what you're referring to :/

**Comment by MrTimscampi** [comment]
@Foggalong That's a screenshot of the Onine Accounts page in Settings in Gnome 3.24.  
The mail icon used for "IMAP and SMTP" isn't properly centered vertically, because it's using "mail-unread-symbolic".

I'm not sure of the order GTK tries to use the icons in the line I gave, but my guess is that currently it uses "mail-unread-symbolic" because nothing else is available ? The most sense would be to make a "mail" icon that would (hopefully) take precedence on that non-centered one.

I can try locally and make a commit using a symlink to "mail-message" if you want ?

**Comment by Foggalong** [comment]
@me4oslav I spent the entire time looking at horizontal alignment, didn't even notice the vertical alignment was out of wack. My back!
